pkgs/development/python-modules/pipx/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, userpath
+, argcomplete
+, packaging
+}:
+
+buildPythonPackage rec {
+  pname = "pipx";
+  version = "0.15.5.0";
+
+  # pypi tarball doesn't contain tests but we have no reason to fetch from github
+  # because tests require venv (https://github.com/pipxproject/pipx/blob/master/tests/helpers.py#L10)
```

Comment:
And why would that be a problem?

Reply:
The problem is i don't know, how to run pytest tests under venv here

Reply:
Tests are run using custom fork of nox (https://github.com/pipxproject/pipx/blob/080fcc947f1a3b51db1c7b6b14dd2a8cbfa1166d/noxfile.py#L7). That's why i am not sure, whether we are able to run it here

Reply:
Currently trying to make it work using `venv`

Reply:
I've disabled tests, which require internet connection and run them under a `venv`

Reply:
I made a video https://www.youtube.com/watch?v=jXd-hkP4xnU which might help with packing python applications within nixpkgs